pkg/webhook/validate.go

```diff
@@ -213,6 +213,15 @@ func staticValidateSriovNetworkNodePolicy(cr *sriovnetworkv1.SriovNetworkNodePol
 	if cr.Spec.DeviceType == consts.DeviceTypeVfioPci && cr.Spec.IsRdma {
 		return false, fmt.Errorf("'deviceType: vfio-pci' conflicts with 'isRdma: true'; Set 'deviceType' to (string)'netdevice' Or Set 'isRdma' to (bool)'false'")
 	}
+	if strings.EqualFold(cr.Spec.LinkType, consts.LinkTypeIB) && !cr.Spec.IsRdma {
```

Comment:
I think this one was removed  no?

https://github.com/k8snetworkplumbingwg/sriov-network-operator/commit/78bde32285684f1300d2c689b974edc9421bbe75

Reply:
Thx! Good catch! done